docs/posts/flink-quickstart/index.qmd

```diff
@@ -0,0 +1,259 @@
+---
+title: "Ibis goes real-time! Introducing the new Flink backend for Ibis"
+author: "Deepyaman Datta"
+date: "2024-01-22"
```

Comment:
let's ship this blog next week -- perhaps Monday?

Reply:
Will update!

Reply:
Updated to Monday